lib/monthly.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
```

Comment:
Service names should use action verbs. The name `MonthlyVatReturnService` could be more descriptive and action-oriented, such as `CalculateMonthlyVatReturnService`.